sdk/daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -125,9 +125,9 @@ object Node {
       stakeholders: Set[Party],
       override val keyOpt: Option[GlobalKeyWithMaintainers],
       override val byKey: Boolean,
+      val interfaceId: Option[TypeConName],
```

Comment:
I would expect, this `interfaceId`, if defined, needs to be included into `packageIds`. Otherwise, you are saying that the engine can produce this FetchNode without loading the interface package.

Even if you conclude that it is correct like this on the engine side, we would have to double-check the Canton side once more.

Reply:
Yes. `interfaceId` should be part of the packageIds`.

Reply:
Added